frontend/src/helpers/vCalendarDragAndDrop.js

```diff
@@ -2,11 +2,22 @@
  * helpers for VCalendar DragAndDrop composables
  */
 
+const ONE_MINUTE = 60 * 1000
+const ONE_HOUR = 60 * ONE_MINUTE
+const ONE_DAY = 24 * ONE_HOUR
+
+const roundToMinutes = 15 // minutes
+const roundToMilliseconds = roundToMinutes * ONE_MINUTE
+
 // helper function to convert Vuetify day & time object into timestamp
 const toTime = (tms) => {
   return new Date(tms.year, tms.month - 1, tms.day, tms.hour, tms.minute).getTime()
 }
 
+const roundTime = (time) => {
```

Comment:
Could we use this function in useDragAndDropMove.js as well, and then remove roundTimeDown entirely?

Reply:
Fixed in #3468